docs/release_notes/8.0.0-1.md

```diff
@@ -0,0 +1,83 @@
+* You can now [unshard a collection](https://www.mongodb.com/docs/manual/tutorial/unshard-collection/#std-label-unshard-collection-task) if it should reside on a single shard or the collection requires resource isolation. You can either specify the desired shard or the collection will be moved onto the shard with the least load. 
+* You can now save costs on additional computing resources when deploying a sharded cluster. The config server replica set can now store both the configuration and the database data. Such a deployment is recommended for small clusters and is especially useful for evaluating the cluster without having to pay extra costs.
+* You can now make self-managed backups in sharded clusters using `mongodump`. Run the `fsync` command with the `lock` field set to `true` from `mongos` to lock each shard and prevent further writes. Then, run `fsynckUnlock` to unlock shards.
+
+#### Logging improvements
+
+You can now configure database profiler to log slow operations based on the time MongoDB actually spent processing that operation. This gives you more accuracy when you analyze performance. 
+
+#### Security improvements
+
+* You can now run range queries on encrypted fields, specifically for numeric data types and dates. This expands the range of use cases for queryable encryption, including accounting operations, healthcare, financial services, and other strictly regulated data operations where data safety is crucial.
+* The logging format in MongoDB is now [OCSF](Open Cybersecurity Schema Framework) compliant. This simplifies integration with external monitoring systems since the log output is compatible out of the box. This saves the time of your developers enabling them to focus on other tasks.
+
+Other improvements include the following:
+
+* [SERVER-76883](https://jira.mongodb.org/browse/SERVER-76883) - Suppressed the "role does not exist" warning log for LDAP group role resolution
+* [SERVER-82221](https://jira.mongodb.org/browse/SERVER-82221) - Included commit-pending metadata entries to the listCollections and listIndexes output
+* [SERVER-94635](https://jira.mongodb.org/browse/SERVER-94635) - Improved handling of logical session refresh by making the session refresh parameters configurable
+* [SERVER-95244](https://jira.mongodb.org/browse/SERVER-95244) - Fixed the `tassert()` failure for upsert operations run when a client connects directly to shard
+* [WT-13409](https://jira.mongodb.org/browse/WT-13409) - Fixed the data corruption issue by properly handling a return value from a function call in transaction checkpoint in WiredTiger
+
+Find the full list of changes in the [MongoDB 8.0.1 Community Edition release notes](https://www.mongodb.com/docs/manual/release-notes/8.0/#8.0.0---oct-2--2024)
```

Comment:
Replace the link with the https://www.mongodb.com/docs/manual/release-notes/8.0/#8.0.1---oct-9--2024